src/sentry/static/sentry/app/views/performance/landing.tsx

```diff
@@ -28,16 +37,18 @@ type Props = {
 type State = {
   eventView: EventView;
   error: string | undefined;
+  currentView: FilterViewType;
 };
 
 class PerformanceLanding extends React.Component<Props, State> {
   static getDerivedStateFromProps(nextProps: Props, prevState: State): State {
     return {...prevState, eventView: generatePerformanceEventView(nextProps.location)};
   }
 
-  state = {
+  state: State = {
     eventView: generatePerformanceEventView(this.props.location),
     error: undefined,
+    currentView: 'ALL_TRANSACTIONS',
```

Comment:
This could be an enum property if the type was also an enum.

Reply:
Done in https://github.com/getsentry/sentry/pull/17954/commits/90c2ea1cb9260ab39a5a6a471f710dfd222f30a8